scripts/integration/docker-compose.nginx.yml

```diff
@@ -0,0 +1,64 @@
+services:
+  squid:
```

Comment:
remind me why we're running both nginx and squid?

Reply:
to check that the proxy feature is working well.
To give you some more details, `nginx-proxy` is only accessible trough the proxy (see the networks), so if the proxy if not well implemented, it should fail.